{"uuid": "19a3a05f-3188-4c5c-8aa9-534659c14cfe", "children": ["e712ef28-5e08-4bf7-b4ae-f281b2d2a7a9"], "befores": [{"name": "with_login_cpc", "status": "broken", "statusDetails": {"message": "KeyError: 240\n", "trace": "  File \"d:\\python37-32\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"d:\\python37-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 927, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"d:\\python37-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 774, in call_fixture_func\n    res = next(it)\n  File \"d:\\coding\\case\\conftest.py\", line 14, in with_login_cpc\n    cpclogin.open_login_url()\n  File \"d:\\coding\\biz\\caipiao_client\\login_page.py\", line 13, in open_login_url\n    login_url = self.URL['login_page_url']['http'] + self.URL['login_page_url']['server'][ip] + self.URL['login_page_url']['base']\n"}, "start": 1568963235334, "stop": 1568963240507}], "start": 1568963235334, "stop": 1568963240533}